### PR TITLE
Add secondary nav and links to anbox-cloud

### LINKED ISSUE #167: Add links to anbox-cloud after pages have been built
These links need adding to the homepage once the pages are live. See [copy doc](https://docs.google.com/document/d/1SfpOBeDJUZj2ZYq4V8lHOUbQF9Yzo7mp5NajJDVwFMI/edit#heading=h.pnlo30ndk43n)

image

image

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Add secondary nav and links to anbox-cloud homepage
- **NOTE**: Learn more about virtual devices link from homepage links to this [copy doc](https://docs.google.com/document/d/18SJoqttKOsDZ0zLO2eqcH6rot7T69TsNzy43nQbDgJs/edit#heading=h.sh0me35kv6j0) - but I can't see this issue tracked anywhere, and this page doesn't exist? Will create an issue to add it when we firgure out what's going on

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Check the secondary nav works as expected
- Check the links against the [copydoc](https://docs.google.com/document/d/1SfpOBeDJUZj2ZYq4V8lHOUbQF9Yzo7mp5NajJDVwFMI/edit#) all there except the `Learn more about virtual devices`

## Issue / Card

Fixes #167 

## Screenshots

Screenshot 2021-06-04 at 14 48 01

### CODE REVIEW

File: static/sass/_pattern_navigation.scss
Comment (steverydz) [inline static/sass/_pattern_navigation.scss:2]: We have a convention where we don't concatenate classes in this way because it makes them harder to search for or keep track of in a large stylesheet.

See: https://canonical-web-and-design.github.io/practices/coding/stylesheets.html#naming-convention

**Review by steverydz — COMMENTED** [review]
It all works fine, but there is a comment about the CSS.

File: static/sass/_pattern_navigation.scss
Comment (bethcollins92) [reply static/sass/_pattern_navigation.scss:2]: Ahh yes I totally forgot about this! Thanks Steve I'll change it now

**Comment by bethcollins92** [comment]
@steverydz Fixed and ready for re-review

File: static/sass/_pattern_navigation.scss
Comment (steverydz) [inline static/sass/_pattern_navigation.scss:44]: not sure this comment is needed

File: static/sass/_pattern_navigation.scss
Comment (steverydz) [inline static/sass/_pattern_navigation.scss:63]: this could be:

```
padding: .55rem 1rem 0;
```

**Review by steverydz — APPROVED** [review]
LGTM 👍 

A couple of minor suggestions in the code

**Comment by bethcollins92** [comment]
Spoken to @morphis and he has asked to not merge this until we work out what's going on with the missing page

**Comment by morphis** [comment]
@bethcollins92 @SirSamTumless I think we can land this if we leave out the telco portion for now and then replace it later with the mobile devices page. Does that sound good?

**Comment by bethcollins92** [comment]
@morphis I've removed the telco section, let me know if this is now ok to merge or if I've misunderstood, thanks!

**Pull request merged**
